definitions/checks/check_for_newer_packages.rb

```diff
@@ -33,9 +33,11 @@ def check_for_package_update
     end
 
     unless packages_with_updates.empty?
-      failure_message = 'An update is available for package(s): '\
-                        "#{packages_with_updates.join(',')}. Please update before proceeding!"
-      fail! failure_message
+      command = ForemanMaintain.pkg_and_cmd_name[1]
+      fail! 'Required updates for some packages detected. '\
+        "Please update to the latest #{@manual_confirmation_version}.z release "\
+        'before proceeding, using: '\
+        "#{command} upgrade run --target-version #{@manual_confirmation_version}.z"
```

Comment:
The message is too long and doesn't look good, request to modify it something like below,

```suggestion
      fail! 'Required updates for some packages detected.'\
        ""\nPlease update to the latest #{@manual_confirmation_version}.z release "\"\
        'before proceeding, using:'\
        "\n# #{command} upgrade run --target-version #{@manual_confirmation_version}.z"
```